packages/venia-ui/.storybook/webpack.config.js

```diff
@@ -39,7 +44,8 @@ module.exports = async ({ config: storybookBaseConfig, mode }) => {
             STORE_VIEW_LOCALE: JSON.stringify(global.LOCALE),
             STORE_VIEW_CODE: process.env.STORE_VIEW_CODE
                 ? JSON.stringify(process.env.STORE_VIEW_CODE)
-                : JSON.stringify(storeConfigData.code)
+                : JSON.stringify(storeConfigData.code),
+            AVAILABLE_STORE_VIEWS: JSON.stringify(availableStores)
```

Comment:
There is also the `venia-concept` storybook config which I think we have so folks can use storybook after scaffolding.